sbnanalysis/ana/SBNOsc/config/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+INSTALL(FILES NumuConfig.fcl
+  DESTINATION
+    $ENV{SBN_ANALYSIS_FHICL_DIR}
```

Comment:
This might not be set before building, safer to base it on the CMake install directory.

Reply:
Fixed.